src/components/IDO/IDO.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { Grid, Typography, Box, CardMedia } from '@material-ui/core'
+
+import useStyle from './style'
+import AmountInput from '@components/Inputs/ExchangeAmountInput/ExchangeAmountInput'
+import AnimatedButton from '@components/AnimatedButton/AnimatedButton'
+import icons from '@static/icons'
+
+interface IDOinterface {
+  xBTC: string
+  xETH: string
+  SOL: string
+  USD: string
+  xUSD: string
+  HEADER: string
+  ButtonHeader: string
+}
+
+const IDO: React.FC<IDOinterface> = ({ HEADER, xUSD, USD, SOL, xETH, xBTC, ButtonHeader }) => {
+  const classes = useStyle()
+
+  return (
+    <Grid className={classes.idoWrapper}>
+      <Grid container className={classes.header}>
+        <Typography component='h1'>IDO</Typography>
+      </Grid>
+      <Grid container className={classes.root} direction='column'>
+        <Box className={classes.depositHeader}>
+          <Typography component='h1'>{HEADER}</Typography>
+        </Box>
+        <Grid className={classes.AmountInputContainer}>
+          <AmountInput
+            disabled={true}
```

Comment:
don't hardcode data inside components if not necessary, we have props and stories for that

Reply:
done